.gitignore

```diff
@@ -1,3 +1,4 @@
 # IDE and OS files
 .DS_Store
 .idea
+.history
```

Comment:
@squallstar Can you confirm if we want to ignore `.history` file?

```suggestion
```

Reply:
@tonytlwu yes it's fine to add `.history` to `.gitignore`, we discussed this in the past, it's a feature on VSCode which basically mimics `.git`